From: Richard B Winters <rik@mmogp.com>
Date: Fri, 29 Jan 2016 10:43:26 -0500
Subject: Fix privacy-breach-generic caused by footer template

 - An image fetched from githubuserconent needs to be 
   replaced with a local cache to avoid the 
   aforementioned lintian informational warning

Change-Id: I3798ca0f774161ed6d99d0da25369ad421e69397
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 doc/website-v1/crm.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/doc/website-v1/crm.conf
+++ b/doc/website-v1/crm.conf
@@ -589,7 +589,7 @@
 </div>
 </div>
 
-<a href="https://github.com/ClusterLabs/crmsh"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png"></a>
+<a href="https://github.com/ClusterLabs/crmsh"><img style="position: absolute; top: 0; right: 0; border: 0;" src="/img/fork-me-on-github.png" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png"></a>
 
 </body>
 </html>
